ui/pages/confirmations/components/confirm/info/batch/batch-simulation-details/batch-simulation-details.tsx

```diff
@@ -63,18 +63,22 @@ export function BatchSimulationDetails() {
     return null;
   }
 
-  const finalBalanceChanges = approveBalanceChanges?.map((change) => ({
-    ...change,
-    onEdit:
-      change.asset.standard === TokenStandard.ERC20
-        ? () => handleEdit(change)
-        : undefined,
-  }));
+  const approveRow: StaticRow[] = useMemo(() => {
```

Comment:
Minor, should we call this `approveRows` now?

Reply:
I updated the PR.